### PR TITLE
Call Of Duty 3 patches

### PR DESCRIPTION
**Pull request by Clippy95**
Mostly patches that fixes graphical artifacts with the game
[Grass Before](https://github.com/user-attachments/assets/d4b3d72b-1545-471d-ac2f-6abae8cd562a)
[Grass After](https://github.com/user-attachments/assets/1de35dfe-6576-44bd-b9ac-0f92e9cd7a5d)

### CODE REVIEW

File: patches/415607E1 - Call Of Duty 3 MP (TU0).patch.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+title_name = "Call Of Duty 3" # MP TU0
+title_id = "415607E1" #AV-2017
```
Comment (illusion0001) [inline patches/415607E1 - Call Of Duty 3 MP (TU0).patch.toml:2]: Need space here
```suggestion
title_id = "415607E1" # AV-2017
```

File: patches/415607E1 - Call Of Duty 3 MP (TU3).patch.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+title_name = "Call Of Duty 3" # MP TU3
+title_id = "415607E1" #AV-2017
```
Comment (illusion0001) [inline patches/415607E1 - Call Of Duty 3 MP (TU3).patch.toml:2]: Need space here
```suggestion
title_id = "415607E1" # AV-2017
```

File: patches/415607E1 - Call Of Duty 3.patch.toml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+title_name = "Call Of Duty 3" # SP
+title_id = "415607E1" #AV-2017
```
Comment (illusion0001) [inline patches/415607E1 - Call Of Duty 3.patch.toml:2]: Need space here
```suggestion
title_id = "415607E1" # AV-2017
```

**Comment by Clippy95** [comment]
Thanks, still not ready to merge as I'm still trying to remove lens flare / FOV patch.

**Comment by alek-san-dr** [comment]
hey @Clippy95 
i've been tinkering with this patch for a few hours and can't figure out how to make it work with the Fr,Es,It version of the game 
seems the address is not the same for the grass (i believe) 
how could i be sure of the address ?

**Pull request merged**